### DISCUSSION TITLE
Hva gjør vi med kode vi ønsker å dele, men som ikke egentlig har noe med designsystemet å gjøre?

### DISCUSSION THREAD

**Discussion by wkillerud** [fremtind/jokul#3305] [Annet]
Spørsmålet dukket opp i #3300 da jeg så en mulighet for å lage pakker for et mellomlag foran Mixpanel SDK.

- [analytics](https://github.com/fremtind/jokul/tree/582d88fe0a8c5427af2d5758bcfe05ae9e32f2b8/packages/analytics)
- [analytics-mixpanel](https://github.com/fremtind/jokul/tree/582d88fe0a8c5427af2d5758bcfe05ae9e32f2b8/packages/analytics-mixpanel)
- [analytics-react](https://github.com/fremtind/jokul/tree/582d88fe0a8c5427af2d5758bcfe05ae9e32f2b8/packages/analytics-react)

Det ble med rette påpekt at [dette vel ikke _egentlig_ hadde noe spesifikt med Jøkul å gjøre](https://github.com/fremtind/jokul/pull/3300#pullrequestreview-1202261874). Det føltes litt rart å skulle ha 100% utility-kode sammen med designsystemet (selv om vi har syndet litt mot dette allerede, for eksempel i `constants-util` og `actions/`).

**Comment by joms** [comment]
Jeg synes denne var veldig vanskelig å ta et konkret standpunkt til, så jeg stemte på det som føltes minst feil :D
